src/audio/Kconfig

```diff
@@ -269,6 +269,13 @@ endmenu # "Downsampling ratios"
 
 endif # COMP_ASRC
 
+config COMP_GENERIC_PROCESSOR
+	bool "Generic processor"
+	default n
+	help
+	  Select for use of any HiFi external codec libraries.
+	  This component is sort of an adapter for any external HiFi codec.
```

Comment:
I don't think we should mention HiFi here. This is just a generic processor and in theory should work with any codec.

Reply:
Yes you are right, will correct this "help message".

Reply:
I guess this is intened for 3rd party binary libraries/codecs since open source components would just be added as source via PRs ? If so , we should state this in the help.

Reply:
yes this component is sort of an adapter between SOF components and 3party libraries as you said.